core/execution/market.go

```diff
@@ -742,6 +742,7 @@ func (m *Market) OnTick(ctx context.Context, t time.Time) bool {
 	m.updateMarketValueProxy()
 	m.updateLiquidityScores()
 	m.updateLiquidityFee(ctx)
+	m.position.FlushPositionEvents(ctx)
```

Comment:
I don't particularly mind, but this does mean we're only sending the position state events on the _next_ tick, so they'll technically be behind 1 block

Reply:
addressed